components/tools/OmeroWeb/omeroweb/webclient/views.py

```diff
@@ -2901,8 +2904,10 @@ def download_annotation(request, annId, conn=None, **kwargs):
     """ Returns the file annotation as an http response for download """
     ann = conn.getObject("Annotation", annId)
```

Comment:
You probably want to include #5816 here. I think that is trying to fix errors coming from testing this PR?
This line will try to find *any* type of annotation (not just "FileAnnotation") but if you try to get any other type of annotation then ```ann.getFileInChunks()``` subsequently fails (raises 500) and you will not have ```conn.close()``` called.